.travis.yml

```diff
@@ -13,11 +13,14 @@ matrix:
   include:
     - php: 7.1
       env: PHPSTAN=1
+           CODING_STANDARD=1
 
 before_script:
   - composer --prefer-source install
   - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.7; fi
+  - if [[ $CODING_STANDARD = 1 ]]; then composer require --dev symplify/easy-coding-standard:^2.2; fi
```

Comment:
I understand that pushing other CS tools can be important, but I'd rather just use `./vendor/bin/phpcs`, which works very well out of the box. For me, that is the go-to solution, as otherwise we just add more work where things go wrong.